src/views/Permissions.vue

```diff
@@ -0,0 +1,298 @@
+        inputs: [{
+          name: "groupName",
+          value: this.currentGroup.groupName
+        }],
+        buttons: [{
+          text: translate("Cancel"),
+          role: "cancel"
+        },
+        {
+          text: translate("Confirm"),
+          handler: async (data: any) => {
+            if (data.groupName) {
+              emitter.emit('presentLoader')
+
+              try {
+                const resp = await UtilService.updateSecurityGroup({
+                  groupId: this.currentGroup.groupId,
+                  groupName: data.groupName
+                })
+
+                if (!hasError(resp)) {
+                  showToast(translate("Security group renamed successfully."))
+                  await this.store.dispatch('util/getSecurityGroups')
+                  await this.store.dispatch('permission/updateCurrentGroup', this.securityGroups.find((group: any) => group.groupId === this.currentGroup.groupId))
+                } else {
+                  throw resp.data
+                }
+              } catch (error) {
+                showToast(translate("Failed to rename security group."))
+                console.error(error)
+              }
+
+              emitter.emit('dismissLoader')
+            }
+          }
+        }]
+      })
+
+      await alert.present()
+    },
+    async getUsersCount() {
+      if(this.securityGroupUsers[this.currentGroup.groupId]) {
+        return;
+      }
+
+      try {
+        const resp = await PermissionService.getSecurityGroupUsers({
+         entityName: "UserLoginAndSecurityGroup",
+          distinct: "Y",
+          noConditionFind: "Y",
+          filterByDate: "Y",
+          viewSize: 100,
+          inputFields: {
+            groupId: this.currentGroup.groupId
+          }
+        })
+
+        if(!hasError(resp)) {
+          this.securityGroupUsers[this.currentGroup.groupId] = resp.data.count
+        } else {
+          throw resp.data;
+        }
+      } catch(err) {
+        console.error(err)
+      }
+    },
+    async openCurrentGroupUsers() {
+      await this.store.dispatch('user/updateQuery', {queryString: '', securityGroup: this.currentGroup.groupId, status: '', hideDisabledUser: true})
+      this.router.replace('find-users')
+    },
+    async downloadCSV() {
+      let finalJSON = [] as any
+
+      if(Object.keys(this.currentGroup).length) {
+        Object.values(this.currentGroupPermissions).map((permission: any) => {
+          finalJSON.push({
+            "Group Id": permission.groupId,
+            "Permission ID": permission.permissionId,
+            "Permission Desc": permission.description,
+            "Member Created Date": DateTime.fromMillis(permission.fromDate).toFormat('dd-MM-yyyy')
+          })
+        })
+      } 
+      else {
+        finalJSON =  await this.downloadAllCSV() 
+      }
+
+      await jsonToCsv(finalJSON, { download: true })
+
+    },
+    async downloadAllCSV() {
```

Comment:
```suggestion
    async downloadCSVForAllPermissions() {
```

Just a suggestion, you can think of some better name as well.